src/app/pages/addCourse/addCourse.component.ts

```diff
@@ -14,17 +14,22 @@ export class AddCourseComponent implements OnInit, OnDestroy {
 
 	public duration: number;
 	public durationFormated: string;
+	public today = new Date().toLocaleDateString();
 
 	public newCourseForm = this.fb.group({
-		title: [''],
-		start: [''],
-		id: [''],
-		duration: [''],
-		desc: ['']
+		title: ['', [Validators.maxLength(50), Validators.required]],
+		start: [this.today, [Validators.maxLength(500), Validators.required]],
```

Comment:
Start date should be in DD/MM/YY format. 
And description should not exceed 500 symbols.